internal/buffer/settings.go

```diff
@@ -25,10 +25,6 @@ func (b *Buffer) SetOptionNative(option string, nativeValue interface{}) error {
 		if err != nil {
 			screen.TermMessage(err)
 		}
-		err = config.InitGlobalSettings()
```

Comment:
Shouldn't we remove `config.InitRuntimeFiles()` and `config.ReadSettings()` as well?

Reply:
At first I though I keep it to reload the environment (RT files and `ft:` settings) belonging to the `filetype`, but now seems to be the right job for the `reload` only.
`config.InitLocalSettings()` alone will not parse the `settings.json` again and just works with the last parsed values.